bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -129,7 +129,7 @@ Thing ipcamera:hikvision:West "West Camera"
 ### Reolink
 
 - NVR's made by Reolink have ONVIF disabled by default and may require a screen connected to the hardware to enable ONVIF or newer firmwares may be able to do this via their app or web UI.
-- This binding will use the Reolink API for polling the alarms if the `nvrChannel` is 1 or higher and does not need ONVIF to be enabled. To use ONVIF event methods for the alarms, you can set `nvrChannel` to 0.
+- This binding will use the Reolink API for polling the alarms. To use Reolink API, set `onvifPort` to 0. Otherwise ONVIF event methods are used for the alarms.
```

Comment:
```suggestion
- This binding will use ONVIF methods to receive alarm events by default. To use polling with the Reolink API, set `onvifPort` to 0 and make sure the `nvrChannel` matches the port your camera is connected to the NVR with.
```

Reply:
This seems to be an improvement. If you can fix this @Skinah thi si ready to be merged. 
Would be nice to get a confirmation from @uselessCookie as author of this PR that he agrees with the changes.